.travis.yml

```diff
@@ -21,7 +21,8 @@ env:
     # Set default python version to avoid repetition later
     - BUILD_DOCS=false
     - PYTHON_VERSION=2.7
-    - MAIN_CMD="python ./testsuite/MDAnalysisTests/mda_nosetests --processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50"
+    - MAIN_CMD="python ./testsuite/MDAnalysisTests/mda_nosetests analysis auxiliary coordinates core formats topology utils
```

Comment:
I think that this is a temporary thing and within one month from now, we will be back to something like `python ./mda_pytest --flags`.
Though I can do as suggested by you, it feels like an overkill to me.